docs/tech/system/backend.md

```diff
@@ -10,12 +10,145 @@ Create diagram of their interaction
 Database structure
 }
 
-
 ## Introduction
 
+The backend system consists of mainly the REST APIs written in Python Django framework to power
+Esamwad, Shiksha Saathi & NLA for Parents Application. It internally uses User Service, Yaus,
+Doc Generator, Templator, ODK Aggregate, etc. as the core building blocks (open source & built
+by Samagra). It also uses various open source 3rd party tools like Hasura, Posthog, Sentry, Fusion
+Auth, Grafana, Vault, Airflow etc.
+
+A brief overview of all the services is shown below:
+
+![System Overview](../../../static/img/samarth-hp-backend-Services.jpg)
+
 ## Repos
 
+- Esamwad Backend: https://github.com/Samarth-HP/esamwad-backend
+- User Service: https://github.com/Samagra-Development/user-service
+- Yaus: https://github.com/Samagra-Development/yaus
+- Doc Generator: https://github.com/Samagra-Development/Doc-Generator
+- Templater: https://github.com/Samagra-Development/templater
+- ODK: https://github.com/samagra-comms/odk
+- ODK UCI Adapter: https://github.com/Samarth-HP/odk-uci-adapter
+- Posthog: https://github.com/Samagra-Development/posthog-deploy
+- Airflow: https://github.com/Samagra-Development/airflow
+
 ## Important modules
 
-### Implementation
+- Assessment Module
+- Student Enrollment Module
+- SLC Module
+- SMS Module
+
+## Implementation
+
+### Assessment Module
+
+There are 4 types of assessments:
+
+- LO(Learning objective)
+- Unit
+- Question
+- None
+
+And each assessment can be evaluated based on 4 types of evaluation parameters on which we assess students'
+submissions:
+
+- Grade
+- Marks
+- Percentage
+- None
+
+The final submission results can be evaluated based on 3 outcomes:
+
+- TSP(Total Studnet Passing)
+- Boolean
+- Marks
+
+There are a total of 32 possible combinations from the above divisions and out of those assessments, 
+we support the Grade and Marks evaluation parameter, LO, Unit, Questions, None assessment type, 
+and TSP outcome.
+
+#### Assessment Creation
```

Comment:
Can we link this to the code base as well? Same for other modules as well.

Reply:
Done.